src/Simulation/Simulation.jl

```diff
@@ -239,7 +239,13 @@ function Grid(sim::Simulation{T, Cylindrical};
 
     world_Δr, world_Δφ, world_Δz = width.(world.intervals)
     world_r_mid = (world.intervals[1].right + world.intervals[1].left)/2
-    
+    if for_weighting_potential && world_Δφ > 0 
+        world_φ_int = SSDInterval{T, :closed, :open, :periodic, :periodic}(0, 2π)
+        world_Δφ = width(world_φ_int)
```

Comment:
Do we **always** want it to be full `2π `? I think this makes the `full_2π` keyword obsolete?

We could set `full_2π = true` here...

Reply:
Yes, I think it is obsolete. It's nowhere used. I will remove it.

I would keep it as we did it before. Always calculate the weigting potentials for 2π. 
Only if it is completly phi-symmetric. Than, also only 2D. 

I think we should refactor the symmetry handling, #215, in general. But for now we just need a quick fix.